### PR TITLE
support the fuzzyQueryColumn and fuzzyQueryEncryptorName for distsql and UT, SIT

### PR DESCRIPTION
**Pull request by gxxiong**
Ref #20435 .

Changes proposed in this pull request:
-Support the fuzzyQueryColumn and fuzzyQueryEncryptorName for distsql and UT, SIT

---

Before committing this PR, I'm sure that I have checked the following options:
- [X] My code follows the [code of conduct](https://shardingsphere.apache.org/community/en/involved/conduct/code/) of this project.
- [X] I have self-reviewed the commit code.
- [ ] I have (or in comment I request) added corresponding labels for the pull request.
- [X] I have passed maven check locally : `mvn clean install -B -T2C -DskipTests -Dmaven.javadoc.skip=true -e`.
- [ ] I have made corresponding changes to the documentation.
- [X] I have added corresponding unit tests for my changes.

### CODE REVIEW

File: features/encrypt/core/src/test/java/org/apache/shardingsphere/encrypt/rule/EncryptRuleTest.java [hunk 1]
```diff
@@ -123,28 +123,47 @@ public void assertFindPlainColumn() {
...
+    @Test
+    public void assertFindFuzzyQueryColumn() {
+        assertFalse(new EncryptRule(createEncryptRuleConfiguration()).findFuzzyQueryColumn("t_encrypt", "pwd_cipher").isPresent());
+    }
+    
+    @Test
+    public void assertGetEncryptFuzzyQueryValues() {
+        List<Object> encryptFuzzyQueryValues = new EncryptRule(createEncryptRuleConfiguration())
+                .getEncryptFuzzyQueryValues(DefaultDatabase.LOGIC_NAME, DefaultDatabase.LOGIC_NAME, "t_encrypt", "pwd", Collections.singletonList(null));
+        for (final Object value : encryptFuzzyQueryValues) {
```
Comment (strongduanmu) [inline features/encrypt/core/src/test/java/org/apache/shardingsphere/encrypt/rule/EncryptRuleTest.java:135]: Please remove final and rename value to each.

File: features/encrypt/core/src/test/java/org/apache/shardingsphere/encrypt/rule/EncryptRuleTest.java [hunk 2]
```diff
@@ -123,28 +123,47 @@ public void assertFindPlainColumn() {
...
+    
     @Test
     public void assertIsQueryWithCipherColumn() {
-        EncryptColumnRuleConfiguration encryptColumnConfig = new EncryptColumnRuleConfiguration("encrypt_column", "encrypt_cipher", "", "", "test_encryptor", null);
+        EncryptColumnRuleConfiguration encryptColumnConfig = new EncryptColumnRuleConfiguration("encrypt_column", "encrypt_cipher", "", "", "", "test_encryptor", null);
         EncryptTableRuleConfiguration tableConfig = new EncryptTableRuleConfiguration("t_encrypt", Collections.singletonList(encryptColumnConfig), null);
         AlgorithmProvidedEncryptRuleConfiguration ruleConfig = new AlgorithmProvidedEncryptRuleConfiguration(
                 Collections.singleton(tableConfig), Collections.singletonMap("test_encryptor", new CoreEncryptAlgorithmFixture()), true);
         EncryptRule actual = new EncryptRule(ruleConfig);
         assertTrue(actual.isQueryWithCipherColumn("t_encrypt", "encrypt_column"));
```
Comment (strongduanmu) [inline features/encrypt/core/src/test/java/org/apache/shardingsphere/encrypt/rule/EncryptRuleTest.java:153]: Please remove useless blank line here.

**Review by strongduanmu — CHANGES_REQUESTED** [review]

File: features/encrypt/core/src/test/java/org/apache/shardingsphere/encrypt/rule/EncryptRuleTest.java (hunk 2 above)
Comment (gxxiong) [reply features/encrypt/core/src/test/java/org/apache/shardingsphere/encrypt/rule/EncryptRuleTest.java]: Problem is fixed

**Review by RaigorJiang — APPROVED** [review]

**Comment by RaigorJiang** [comment]
> 



> LGTM, @RaigorJiang Can you help double check?

Good implementation, merged!

**Pull request merged**